src/sugar3/speech.py

```diff
@@ -220,6 +222,28 @@ def __init__(self, **kwargs):
         self._save_timeout_id = -1
         self.restore()
 
+    def _marked_messages_cb(self, player, value):
+        '''
+        This method provides access to mark message values which can be used
+        for text highlighting.(see readetexts activity)
+
+        In your Speech Toolbar class, connect the `mark` signal to your
+        own callback.
+
+        .. code-block:: python
+
+            from sugar3.speech import SpeechManager
+
+            class SpeechToolbar(Gtk.Toolbar):
```

Comment:
Not all uses of SpeechManager require a SpeechToolbar, so mentioning SpeechToolbar seems overly specific.

Reply:
Yes you're right, I made an error of thinking of a particular case. I'll fix that, thanks.